LoRaEngine/test/LoRaWanNetworkServer.Test/ADRTest.cs

```diff
@@ -137,21 +137,21 @@ public async Task Perform_Rate_Adapatation_When_Possible(uint deviceId, int coun
         // DR0 with high lsnr will result in move to DR5
         [InlineData(2, 20, "SF12BW125", 5, 7)]
         // DR1 with high lsnr will result in move to DR5
-        [InlineData(3, 20, "SF11BW125", 5, 7)]
+        // [InlineData(3, 20, "SF11BW125", 5, 7)]
```

Comment:
Was it intentional to remove the commented test cases?

Reply:
No. Enabled them again. Thanks.